tasks/Node/README.md

```diff
@@ -0,0 +1,88 @@
+# Azure Pipelines DevTest Labs Tasks
+
+Collection of build/release tasks to interact with [Azure DevTest Labs](https://azure.microsoft.com/en-us/services/devtest-lab/).
+
+# References
+
+* [Azure Pipelines Tasks](https://github.com/microsoft/azure-pipelines-tasks) - source code for out of the box tasks provided with [Azure DevOps](https://azure.microsoft.com/en-ca/services/devops/).
+* Article [Add a build or release task](https://docs.microsoft.com/en-us/azure/devops/extend/develop/add-build-task?view=azure-devops&viewFallbackFrom=vsts).
+
+# Release Notes
+
+See the [RELEASENOTES.md](RELEASENOTES.md) file for details.
+
+# Pre-requisites
+
+The following are tools used to create these tasks and are recommended.
+
+* The latest version of [Visual Studio Code](https://code.visualstudio.com/).
+* The latest version of [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
+* The latest version of [Node.js](https://nodejs.org/en/download/).
+* [Typescript Compiler](https://www.npmjs.com/package/typescript) v2.2.0 or later.
```

Comment:
Not sure if you need to keep this line about the TS compiler in the ReadMe. It should be a dependency in your package.json and it gets invoked by the webpack ts-loader so its not used directly.

Reply:
I listed it to be in line with what is written for tasks in general. I think we may be one of the few that use Webpack to package tasks.

When the user want to compile things for testing they may run `tsc` directly. But you're correct about the `package.json`. It could be removed from the `README.md`.